### DISCUSSION TITLE
Make default path to pdftotext dynamic?

### DISCUSSION THREAD

**Discussion by robin7331** [spatie/pdf-to-text#35] [Ideas]
I've installed pdftotext through `homebrew` on my macOS Catalina 10.15.6.
The binary ended up being in `/usr/local/bin/pdftotext` rather than in your default path `/usr/bin/pdftotext`.

I know there is an option to set a specific binary path but what if this library would default to the output of `which pdftotext` ?

**Comment by iec989** [comment]
This would be awesome - it would be great if there was a config file where you could input the path, so yoi can manage multiple environments
